WalletWasabi/WabiSabi/Client/BaseKeyChain.cs

```diff
@@ -31,7 +31,8 @@ public OwnershipProof GetOwnershipProof(IDestination destination, CoinJoinInputC
 		var ownershipProof = OwnershipProof.GenerateCoinJoinInputProof(
 			signingKey,
 			new OwnershipIdentifier(identificationKey, destination.ScriptPubKey),
-			commitmentData);
+			commitmentData,
+			ScriptPubKeyType.TaprootBIP86);
```

Comment:
With this change wasabi doesn't work because currently it only works with Segwit.

Reply:
Ohhh this is also why tests fail and don't finish.

Reply:
Sorry, my fault.

Fixed in https://github.com/zkSNACKs/WalletWasabi/pull/8831/commits/3670f1755605492d16aa55e5c788fb0407c0a0a2.